### PR TITLE
make compatible with Elasticsearch 6.x

### PR DESCRIPTION
**Pull request by eynand**
Hi,
I've noticed an error when tried to work with Elasticsearch 6.6.1:
1. mapping is not compatible to the new versions of ES - corrected the mapping
2. theres is not type anymore in ES indices - change the doc type to the defualt.

Thanks,
Eynan

### CODE REVIEW

File: newsplease/pipeline/pipelines.py [hunk 1]
```diff
@@ -452,10 +452,10 @@ def __init__(self):
             # check if the necessary indices exist and create them if needed
             if not self.es.indices.exists(self.index_current):
                 self.es.indices.create(index=self.index_current, ignore=[400, 404])
-                self.es.indices.put_mapping(index=self.index_current, doc_type='article', body=self.mapping)
+                self.es.indices.put_mapping(index=self.index_current, doc_type='_doc', body=self.mapping)
```
Comment (fhamborg) [inline newsplease/pipeline/pipelines.py]: why did you change from article to _doc?

Comment (eynand) [reply newsplease/pipeline/pipelines.py]: So, this is not a must, but according to : https://github.com/elastic/elasticsearch/pull/27816
It’s the right way to go.

**Comment by fhamborg** [comment]
Thank you! :-)

**Comment by eynand** [comment]
Thank you for this great tool!

‫ב-10 במרץ 2019, בשעה 13:53, ‏‏Felix Hamborg ‏<notifications@github.com<mailto:notifications@github.com>> כתב/ה:‬


Thank you! :-)
